### PR TITLE
Update new buttons in Astro and Docz

### PR DESCRIPTION
**Pull request by larimaza**
Update buttons CSS and Docz page, according to new design specs.
We just left out the icon buttons, because we don't have any icon specs yet.

### CODE REVIEW

File: docs/buttons.mdx [hunk 1]
```diff
@@ -19,29 +19,83 @@ Mouseover and click to view `:hover` and `:active` states.
...
-    primary disabled
+    disabled
   </button>
 </Playground>
 
-## colors
+## outline
 
-Buttons can have different colors simply by changing their CSS color class.
+Outline buttons can be used as secondary roles. Secondary buttons present secondary actions when there's already a primary action in the same section or screen.
```
Comment (igorrfc) [inline docs/buttons.mdx]: Reviewing and learning, this is bliss haha

File: docs/buttons.mdx [hunk 2]
```diff
@@ -19,29 +19,83 @@ Mouseover and click to view `:hover` and `:active` states.
...
+  <button className="btn btn--outline-venus btn--medium" disabled>
+    outline disabled
   </button>
+</Playground>
 
-  <button className="btn btn--earth btn--medium">
-    earth
+## ghost buttons
+
+Ghost buttons are visually similar to inline links, but with the size and behavior of regular buttons.
```
Comment (igorrfc) [inline docs/buttons.mdx]: Conceptual suggestion: would be great if we have an example about how to use a ghost button or a link (following the outline button's description). I miss something like that, maybe just me

File: docs/buttons.mdx [hunk 3]
```diff
@@ -63,49 +117,35 @@ Buttons can have different sizes, also by changing their CSS size class.
...
-## outline
-
-Outline buttons can be used as secondary roles. Secondary buttons present secondary actions when there's already a primary action in the same section or screen.
-
-Please note that there are color variations to the outline CSS classes. You also need to specify their size with one of the size CSS classes.
-
 <Playground>
   <button className="btn btn--outline-uranus btn--small">
-    outline uranus
+    outline small
```
Comment (igorrfc) [inline docs/buttons.mdx]: Good one

File: src/css/buttons.css [hunk 4]
```diff
@@ -88,52 +106,178 @@
...
+.btn--ghost-mars:disabled,
+.btn--ghost-uranus.disabled,
+.btn--ghost-venus.disabled,
+.btn--ghost-earth.disabled,
+.btn--ghost-mars.disabled {
+  cursor: not-allowed;
+  background-color: var(--color-space-100);
+  border-color: var(--color-space-100);
+  color: var(--color-moon-200);
+}
```
Comment (igorrfc) [inline src/css/buttons.css]: Could we put an empty line at the end of this file just to avoid this annoying guy here? 😂

Comment (larimaza) [reply src/css/buttons.css]: Done! df4d2b7

File: docs/buttons.mdx (hunk 2 above)
Comment (larimaza) [reply docs/buttons.mdx]: Good suggestion - done! df4d2b7

**Pull request merged**